### PR TITLE
[Feature-1285][client] Add Column Lineage in Flink 1.11,1.12,1.13

### LINKED ISSUE #1285: [Feature][client] Add Column Lineage in Flink 1.11,1.12,1.13
### Search before asking

- [X] I had searched in the [issues](https://github.com/DataLinkDC/dlink/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

_No response_

### Use case

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by aiwenmo**
## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

### CODE REVIEW

File: dlink-client/dlink-client-1.11/src/main/java/com/dlink/executor/CustomTableEnvironmentImpl.java [hunk 1]
```diff
@@ -311,45 +320,46 @@ public <T> void createTemporaryView(String path, DataStream<T> dataStream, Strin
 
     @Override
     public List<LineageRel> getLineage(String statement) {
-        return null;
+        LineageContext lineageContext = new LineageContext(flinkChainedProgram, this);
```
Comment (leechor) [inline dlink-client/dlink-client-1.11/src/main/java/com/dlink/executor/CustomTableEnvironmentImpl.java:323]: can this method as a CustomTableEnvironment's default method implementation? it look like all flink version had the same implementation.

File: dlink-flink/dlink-flink-1.16/pom.xml [hunk 2]
```diff
@@ -28,7 +28,7 @@
         </dependency>
         <dependency>
             <groupId>org.apache.flink</groupId>
-            <artifactId>flink-table-planner_${scala.binary.version}</artifactId>
+            <artifactId>flink-table-planner_2.12</artifactId>
```
Comment (leechor) [inline dlink-flink/dlink-flink-1.16/pom.xml:31]: what fix this version here?

Comment (aiwenmo) [reply dlink-flink/dlink-flink-1.16/pom.xml:31]: Flink 1.16 only has scala 2.12 version.

Comment (leechor) [reply dlink-flink/dlink-flink-1.16/pom.xml:31]: > Flink 1.16 only has scala 2.12 version.

get it. no problem.

File: dlink-client/dlink-client-1.11/src/main/java/com/dlink/executor/CustomTableEnvironmentImpl.java (hunk 1 above)
Comment (aiwenmo) [reply dlink-client/dlink-client-1.11/src/main/java/com/dlink/executor/CustomTableEnvironmentImpl.java:323]: It's difficult to use this method as a CustomTableEnvironment's default method implementation.

**Pull request merged**